### PR TITLE
Update README.md to flow more nicely

### PR DESCRIPTION
**Pull request by skenmy**
Very small reflow of the documentation around the AWS EBS volume example - it just didn't read very well and I found it hard to follow.

### CODE REVIEW

**Comment by skenmy** [comment]
I have signed the CLA

**Comment by ashcrow** [comment]
Shippable build error is a timeout. Unrelated.

Looks good to me.

**Comment by fejta** [comment]
This PR has had no meaningful activity for multiple months. If it is still valid please rebase, push a new commit and reopen the PR. Thanks!
